app/assets/javascripts/initializers/initializeNavigation.js

```diff
@@ -0,0 +1,22 @@
+function listenHamburgerTriggers() {
```

Comment:
refactor (Nick exploring): Since this is all new frontend code, I'm going to see what it'd look like if we import it into a webpacker pack file because our base.js will only continue to grow.